v21.1/known-limitations.md

```diff
@@ -6,6 +6,159 @@ toc: true
 
 This page describes newly identified limitations in the CockroachDB {{page.release_info.version}} release as well as unresolved limitations identified in earlier releases.
 
+## New limitations
+
+### CockroachDB does not properly optimize some left and inverted joins with inverted indexes
+
+[Left joins](joins.html#left-outer-joins) and [inverted joins](joins.html#inverted-joins) involving [`JSONB`](jsonb.html), [`ARRAY`](array.html), or [spatial-typed](spatial-data.html) columns with a multi-column or [partitioned](partition-by.html) [inverted index](inverted-indexes.html) will not take advantage of the index if the prefix columns of the index are unconstrained, or if they are constrained to multiple, constant values.
```

Comment:
nit: this should say left joins and anti joins (it's true the relevant algorithm is inverted join, but the limitation is only for left and anti joins, not inner or semi joins).